ml/src/pairwise_potential.cpp

```diff
@@ -39,27 +39,27 @@
 
 #include <pcl/ml/pairwise_potential.h>
 
-//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-pcl::PairwisePotential::PairwisePotential (const std::vector<float> &feature, 
-                                           const int feature_dimension, 
-                                           const int N, const float w) :
-  N_ (N), w_ (w)
-{  
-  //lattice_.init (feature, feature_dimension, N);
+pcl::PairwisePotential::PairwisePotential(const std::vector<float>& feature,
+                                          const int feature_dimension,
+                                          const int N,
+                                          const float w)
+: N_(N), w_(w)
+{
+  // lattice_.init (feature, feature_dimension, N);
   std::cout << "0---------" << std::endl;
```

Comment:
This looks like debug code that should be removed

Reply:
God knows. I'll comment them out?